src/MonoTorrent/MonoTorrent.Client.Tracker/TrackerTier.cs

```diff
@@ -216,6 +216,8 @@ internal async ReusableTask ScrapeAsync (ScrapeParameters args, CancellationToke
 
             for (int i = 0; i < Trackers.Count; i++) {
                 var tracker = Trackers[(ActiveTrackerIndex + i) % Trackers.Count];
+                if (false == tracker.CanScrape)
```

Comment:
```suggestion
                if (!tracker.CanScrape)
```

It's a bit more idiomatic to write it this way :)